src/qt/peertablesortproxy.cpp

```diff
@@ -26,6 +26,8 @@ bool PeerTableSortProxy::lessThan(const QModelIndex& left_index, const QModelInd
         return left_stats.nodeid < right_stats.nodeid;
     case PeerTableModel::Address:
         return left_stats.addrName.compare(right_stats.addrName) < 0;
+    case PeerTableModel::Direction:
+        return left_stats.fInbound > right_stats.fInbound;
```

Comment:
if you retouch, the comment at https://github.com/bitcoin-core/gui/pull/289/commits/678b57227e5c163cfbe77444d2c8b39c357678c2#diff-dafd60d44b87b3de8fc005415f63356de4fdd1fa772d1c1131a2c8b58c8b1d9aR30 seemed helpful to describe why `>` rather than `<` operator is used here, unlike the other columns

Reply:
Thanks for the review, Addressed in 0183415